components/ws-manager/pkg/manager/monitor.go

```diff
@@ -1244,13 +1230,13 @@ func (m *Monitor) finalizeWorkspaceContent(ctx context.Context, wso *workspaceOb
 		BackupComplete: true,
 		GitStatus:      gitStatus,
 	}
+
+	if doBackup {
```

Comment:
We need to consider `doSnapshot` as well. @sagor999 Correct?
```go
if doBackup || doSnapshot {
```

Reply:
yeah, you are right!

Reply:
Thanks for catching that, I've added `doSnapshot` in the conditional 🙂